clippy_lints/src/loops/mod.rs

```diff
@@ -577,6 +578,33 @@ declare_clippy_lint! {
     "manual implementation of `Iterator::find`"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks for `for (_, v) in a.iter().enumerate()`
```

Comment:
A little bit more textual description would be better. We usually leave the code for the example, if the code is a just a little bit complex.

```suggestion
    /// Checks for uses of the `enumerate` method where the index is unused (`_`)
```

Or maybe a different wording